asm2vec/binary_to_asm.py

```diff
@@ -52,7 +52,9 @@ def _fn_to_asm(pdf: dict | None, asm_minlen: int) -> str:
     """
     if pdf is None:
         return ''
-    if len(pdf['ops']) < asm_minlen:
+    if len(pdf.get('ops', [])) < asm_minlen:
+        if 'ops' not in pdf:
+            logging.info("KeyError: 'ops' not in pdf")
         return ''
     if 'invalid' in [op['type'] for op in pdf['ops']]:
```

Comment:
worth checking that `type` is in op?

```suggestion
    if 'invalid' in [op.get('type', '') for op in pdf['ops']]:
```

Reply:
I haven't encountered such a KeyError in all the 80000 + disassembled binaries, but it wouldn't hurt to think ahead